kv/memberlist/memberlist_client.go

```diff
@@ -427,7 +427,7 @@ func (m *KV) buildMemberlistConfig() (*memberlist.Config, error) {
 	mlCfg.ProbeInterval = 5 * time.Second // Probe a random node every this interval. This setting is also the total timeout for the direct + indirect probes.
 	mlCfg.ProbeTimeout = 2 * time.Second  // Timeout for the direct probe.
 
-	level.Info(m.logger).Log("msg", "Using memberlist cluster label %q and node name %q", mlCfg.Label, mlCfg.Name)
+	level.Info(m.logger).Log("msg", fmt.Sprintf("Using memberlist cluster label %q and node name %q", mlCfg.Label, mlCfg.Name))
```

Comment:
nit: since we're already using go-kit logging, we could avoid the string formatting by doing something like
```suggestion
	level.Info(m.logger).Log("msg", "Using memberlist cluster label and node name", "cluster_label", mlCfg.Label, "node", mlCfg.Name))
```

Reply:
Sure, don't feel strong about it.